### PR TITLE
chore(update): Adapt to latest centreon-ui changes

### PR DESCRIPTION
**Pull request by bdauria**
## Description
The latest centreon-ui build brings some breaking changes. This PR address them.

It also adds a translation for the label "No results found" in the Resources listing. 

## Target serie

- [ ] 19.04.x
- [ ] 19.10.x
- [x] 20.04.x
- [x] 20.10.x (master)

### CODE REVIEW

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -15397,4 +15397,7 @@ msgid "Protocol %s is not allowed"
 msgstr "Le protocole %s n'est pas autorisé"
 
 msgid "The port can only be between 0 and 65535 inclusive"
-msgstr "Le port ne peut être compris qu'entre 0 et 65535 inclus"
\ No newline at end of file
+msgstr "Le port ne peut être compris qu'entre 0 et 65535 inclus"
+
+msgid "No results found"
+msgstr "Pas de résultat trouvé"
```
Comment (kduret) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:15403]:
```suggestion
msgstr "Aucun résultat trouvé"
```

File: www/front_src/src/Resources/Actions/Resource/Acknowledge/Dialog.tsx [hunk 2]
```diff
@@ -82,8 +82,7 @@ const DialogAcknowledge = ({
             label={labelComment}
             fullWidth
             rows={3}
-            error={errors?.comment !== undefined}
-            helperText={errors?.comment}
```
Comment (kduret) [inline www/front_src/src/Resources/Actions/Resource/Acknowledge/Dialog.tsx:86]: helper text is directly manage by TextField component ?

Comment (bdauria) [reply www/front_src/src/Resources/Actions/Resource/Acknowledge/Dialog.tsx:86]: Yes, this was done in the latest centreon-ui PR, error is now either a string or undefined which simplifies its use.

**Comment by jdelpierre** [comment]
I have tested it, so it's ok for me of the translation FR if no result founded on event view listing
image

**Pull request merged**